presets/quintel/SER_energieakkoord_2020.ad

```diff
@@ -185,12 +194,3 @@
 - user_values.transport_useful_demand_trains = 0.7
 - user_values.transport_useful_demand_trains_efficiency = 1.5
 - user_values.transport_useful_demand_truck_kms = 0.7
-- user_values.policy_sustainability_renewable_percentage = 14.0
```

Comment:
This line (and the 10 following it) are removed. Why is that? Don't they exist anymore? Or are they the default values?